subprojects/precondition-tester/build.gradle.kts

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id("gradlebuild.internal.java")
+}
+
+description = "Internal project testing and collecting information about all the test preconditions."
+
+// Special configuration containing all the precondition classes
+val precondition by configurations.creating {
```

Comment:
🤔 Instead of creating a new configuration, could we add the dependencies to `testFixtures` and that would work the same way?

Reply:
❓ Does `testFixtures` getting inherited to all the testing configurations?
I had a real issue trying to declare dependencies (in a tidy way) to all the testing configurations.

Reply:
Actually worked out!